lib/cable_ready/channel.rb

```diff
@@ -131,7 +131,14 @@ def clear
     def broadcast
       operations.select! { |_, list| list.present? }
       operations.deep_transform_keys! { |key| key.to_s.camelize(:lower) }
-      ActionCable.server.broadcast name, "cableReady" => true, "operations" => operations
+      ActionCable.server.broadcast identifier, "cableReady" => true, "operations" => operations
+      clear
+    end
+
+    def broadcast_to(model)
+      operations.select! { |_, list| list.present? }
+      operations.deep_transform_keys! { |key| key.to_s.camelize(:lower) }
+      identifier.broadcast_to model, "cableReady" => true, "operations" => operations
```

Comment:
This doesn't work. It throws an error :

```sh
undefined method `broadcast_to' for "DataChannel":String
```

Reply:
call to `constantize` is missing

Reply:
I believe that you're trying to pass a string as the identifier. If you consult the example above, you'll see that this will generate an error.

I am going to make it raise a more literal exception, so that this is less mysterious. In the meantime, please try calling `cable_ready[DataChannel].broadcast_to(resource)` instead of `cable_ready["DataChannel"].broadcast_to(resource)`.